packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -355,6 +362,178 @@ class Actions extends InheritedWidget {
   }
 }
 
+/// A widget that combines the functionality of [Actions], [Shortcuts],
+/// [MouseRegion] and a [Focus] widget to create a detector that defines actions
+/// and key bindings, and provides callbacks for handling focus and hover
+/// highlights.
+///
+/// This widget can be used to give a control the required detection modes for
```

Comment:
"... can be used to give a control the required ..."

Something seems to be missing here...